docs/LoRA.md

```diff
@@ -121,72 +170,151 @@ if config.merged_hf_save_path is not None:
+with your own Hugging Face Hub username and repo name. (You can also just remove it if you don't want to upload
+the merged checkpoint.)
 
-## The Configuration File
+## Evaluating the Model with HELM
 
-The configuration files are almost identical to the non-LoRA versions, with two differences:
+As we said before, we're going to use HELM to evaluate the model. HELM makes it easy to evaluate models on the
+(many) benchmarks that the HELM team has added to the system. It also makes it easy to compare models to each other,
+and to see the individual results for error analysis.
 
-1. We should dial up the learning rate. We use 3e-4, which is pretty high relative to what's usually recommended (1e-4),
-but it seems to work.
-2. For the LLama-2 version, we set `per_device_parallelism` to 4 (The max for a v3-32 with batch size 128 is 4 since 32 * 4 = 128).
+### 1. Install HELM
 
-We also add WandB tags, but that's not important.
+We recommend installing HELM in a separate virtual environment or conda environment, to minimize the chance of
+dependency conflicts.
 
-You can change the LoRA parameters by adding a `lora` section, or providing appropriate command line flags:
+```bash
+virtualenv -p python3.8 helm
+source helm/bin/activate
+pip install crfm-helm
+```
 
-```yaml
-lora:
-  r: 16
-  dropout: 0.1
+### 2. Evaluate the model
+
+```bash
+export MYMODEL="<somewhere>/gsm8k-llama2-merged"
+helm-run --run-specs gsm:model=$MYMODEL     --enable-huggingface-models $MYMODEL  --suite v1  --max-eval-instances 1000
 ```
 
-The default configs are available as [`alpaca-lora.yaml`](https://github.com/stanford-crfm/levanter/blob/main/examples/alpaca-lora/alpaca-lora.yaml)
-and [`alpaca-lora-llama2.yaml`](https://github.com/stanford-crfm/levanter/blob/main/examples/alpaca-lora/alpaca-lora-llama2.yaml)
+This will run the evaluation.
+If you want to also evaluate the baseline Llama 2 model, you can do that by replacing `$MYMODEL` with
+`meta-llama/Llama-2-7b-hf`:
 
+```bash
+export HUGGING_FACE_HUB_TOKEN=${YOUR TOKEN HERE}
+export MYMODEL="meta-llama/Llama-2-7b-hf"
+helm-run --run-specs gsm:model=$MYMODEL     --enable-huggingface-models $MYMODEL  --suite v1  --max-eval-instances 1000
+```
 
-## Running Alpaca-Lora
+### 3. Summarize the results
 
-Running this modified script is basically identical to running the original script.
-For example:
+This command will summarize the results and produce some nice tables.
 
 ```bash
-infra/babysit-tpu-vm.sh llama-32 -z us-east1-d -t v3-32 --preemptible -- \
-WANDB_API_KEY=${YOUR TOKEN HERE} \
-HUGGING_FACE_HUB_TOKEN=${YOUR TOKEN HERE} \
-bash levanter/infra/run.sh python \
-levanter/examples/alpaca-lora/alpaca_lora.py \
---config_path levanter/examples/alpaca-lora/alpaca-lora-llama2.yaml \
---trainer.checkpointer.base_path gs://<somewhere> \
---hf_save_path gs://<somewhere>
+helm-summarize --suite v1
+```
+
+If you want, you can now look at the LaTeX table:
```

Comment:
I think we should just have them run `helm-run` and `helm-summarize`  and go to the website, and not bother with the latex.  And of course point them to the HELM docs.